packages/server/graphql/public/types/NotifyTeamsLimitReminder.ts

```diff
@@ -0,0 +1,10 @@
+import {NotifyTeamsLimitReminderResolvers} from '../resolverTypes'
+
+const NotifyTeamsLimitReminder: NotifyTeamsLimitReminderResolvers = {
+  __isTypeOf: ({type}) => type === 'TEAMS_LIMIT_REMINDER',
+  organization: ({orgId}, _args, {dataLoader}) => {
```

Comment:
-1 I have slight concerns here: We're not checking organization membership here and in fact once the notification is sent, I can remove the user from the org and they will still see the organization name, meaning they could have access to the org through this member.
Because we're only allowing predefined queries, it's not that big of a deal, but we should set a good example here:
* add a rule in https://github.com/ParabolInc/parabol/blob/83a4a39fb5fab61787ee3e432d6e75d7bec37982/packages/server/graphql/public/permissions.ts#L25 for this field
* (optional) modify `removeOrgUser` to also remove notifications with the `orgId`

Reply:
I think we can do sth. like
```
Organization: {
  '*': or(isSuperUser, isViewerOrgMember)
}
```
and create the `isViewerOrgMember` rule.

Because that's not an issue right now, we can also do it in a separate ticket, but should add it to this sprint.

Reply:
@Dschoordsch What if we store all necessary information for notification as hardcoded values inside a notification object?
I think we can consider notifications as text messages which should not change dynamically.

Reply:
Fair enough. Then an organization name change would not be reflected, but I think that's not a real life issue